website/server/models/task.js

```diff
@@ -71,6 +71,13 @@ export let TaskSchema = new Schema({
     taskId: {type: String, ref: 'Task', validate: [validator.isUUID, 'Invalid uuid.']},
   },
 
+  requiresApproval: {type: Boolean, default: false},
```

Comment:
@paglias Can I get your thoughts here. I wanted to either put these properties under group or under a approval object. However, I am having a trouble allow a user to set sub documents via the `user.post('tasks')` route. For example:
```
task = await user.post(`/tasks/group/${guild._id}`, {
      text: 'test todo',
      type: 'todo',
      'group.requiresApproval': true,
    });
```
Will not work, even if I remove group from the noSet. Do you have any idea why?